cupy/core/include/cupy/cuComplex_bridge.h

```diff
@@ -0,0 +1,34 @@
+/* 
+   This header is to support the "translate_cucomplex" option
+   that turns cuComplex function calls to their Thrust counterparts.
+*/ 
+
+/* ------------------- single complex ------------------- */
+#define cuFloatComplex                complex<float>
+#define cuComplex                     complex<float>
+#define cuCrealf                      real
+#define cuCimagf                      imag
+#define make_cuFloatComplex(A, B)     complex<float>(A, B)
+#define make_cuComplex(A, B)          complex<float>(A, B)
+#define cuConjf                       conj
+#define cuCaddf(A, B)                 (A + B)
+#define cuCsubf(A, B)                 (A - B)
+#define cuCmulf(A, B)                 (A * B)
+#define cuCdivf(A, B)                 (A / B)
+#define cuCabsf                       abs
+#define cuComplexDoubleToFloat        complex<float>
+#define cuCfmaf(A, B, C)              (A * B + C)
+
+/* ------------------- double complex ------------------- */
+#define cuDoubleComplex               complex<double>
+#define cuCreal                       real
+#define cuCimag                       imag
+#define make_cuDoubleComplex(A, B)    complex<double>(A, B)
+#define cuConj                        conj
+#define cuCadd(A, B)                  (A + B)
+#define cuCsub(A, B)                  (A - B)
+#define cuCmul(A, B)                  (A * B)
+#define cuCdiv(A, B)                  (A / B)
+#define cuCabs                        abs
```

Comment:
This seems uncovered by test. Is it difficult to cover all functions?
(I haven't checked if all functions are covered in the test code; this is just one I noticed)

Reply:
Not difficult. I was just lazy. I think I cover all functions now.